ballerina/error.bal

```diff
@@ -14,27 +14,18 @@
 // specific language governing permissions and limitations
 // under the License.
 
-public type GenerateRequestFailed distinct error;
+public type Error distinct error;
```

Comment:
shall we add doc comments for all these error types

Reply:
Fixed in [a92506e](https://github.com/ballerina-platform/module-ballerinax-aws.sns/pull/40/commits/a92506eb6d634ab65d3d1ac8f4bc9313dbc6a1f9)